index.html

```diff
@@ -230,6 +230,52 @@ <h2>About me</h2>
       </div>
     </div>
   </section>
+  <footer>
+    <div class="footer-section">
+    <div>
+      <p class="footer-p">
+        I'm always interested in hearing about new projects,
+         so if you'd like to chat please get in touch.
+      </p>
+    </div>
+   <div>
+    <form action="https://formspree.io/f/mwkzggnv" method="post">
+      <input type="text" placeholder="Full name" maxlength="30" required>
+      <input type="EMAIL" placeholder="Email address" maxlength="30" required>
+      <textarea name="" id="" cols="10" rows="5" required maxlength="500" placeholder="Enter text here"></textarea>
+      <a href="#9" class="Multi-Post-project button form-button">Get in touch</a>
```

Comment:
- [ ] Please you were making changes as requested by the previous code reviewer, your form is not submitting after clicking on the `get in touch` button. I think it is because you use `a` element instead of `<input type="submit" value="Get it touch" />` or button tag